performant-software/textlab
# Fix CSS Compile issue on Startup

**What does this PR do?**
Closes #168 

Changes the configuration for production to use static assets instead of invoking the development fallback process. Adds a note to Readme under "DEPLOYMENT" that explains you need to run rake precompile if you change assets. This will prevent users from encountering (any) CSS compilation errors. 

**How to Test?**
Go to https://mel-textlab-pr-173.herokuapp.com/
Login and verify the CSS error isn't seen

Restart the dyno (via Heroku)
Repeat login procedure and verify the CSS error is still not seen.
